src/osx/pen.cpp

```diff
@@ -98,45 +109,28 @@ wxPen::~wxPen()
 // Should implement Create
 wxPen::wxPen(const wxColour& col, int Width, wxPenStyle Style)
 {
-    m_refData = new wxPenRefData;
-
-    M_PENDATA->m_colour = col;
-    M_PENDATA->m_width = Width;
-    M_PENDATA->m_style = Style;
-    M_PENDATA->m_join = wxJOIN_ROUND ;
-    M_PENDATA->m_cap = wxCAP_ROUND ;
-    M_PENDATA->m_nbDash = 0 ;
-    M_PENDATA->m_dash = 0 ;
+    m_refData = new wxPenRefData(wxPenInfo(col, Width).Style(Style));
 
     RealizeResource();
 }
 
 wxPen::wxPen(const wxColour& col, int Width, int Style)
 {
-    m_refData = new wxPenRefData;
-
-    M_PENDATA->m_colour = col;
-    M_PENDATA->m_width = Width;
-    M_PENDATA->m_style = (wxPenStyle)Style;
-    M_PENDATA->m_join = wxJOIN_ROUND ;
-    M_PENDATA->m_cap = wxCAP_ROUND ;
-    M_PENDATA->m_nbDash = 0 ;
-    M_PENDATA->m_dash = 0 ;
+    m_refData = new wxPenRefData(wxPenInfo(col, Width).Style((wxPenStyle)style));
```

Comment:
s/style/Style

Reply:
Oops, thanks for noticing this, fixed now.